frontend/packages/operator-lifecycle-manager/src/components/create-operand.tsx

```diff
@@ -743,54 +704,113 @@ export const CreateOperandForm: React.FC<CreateOperandFormProps> = (props) => {
-              </label>
-              <div className="co-operand-field-group">
-                {fields
-                  .filter((f) => f.capabilities.includes(group))
-                  .filter((f) => !_.isNil(inputFor(f)))
-                  .map((field) => (
-                    <div key={field.path}>
-                      <div className="form-group co-create-operand__form-group">
-                        <label
-                          className={classNames('form-label', { 'co-required': field.required })}
-                          htmlFor={field.path}
-                        >
-                          {field.displayName}
-                        </label>
-                        {inputFor(field)}
-                        {field.description && (
-                          <span id={`${field.path}__description`} className="help-block text-muted">
-                            {field.description}
-                          </span>
-                        )}
-                        {formErrors[field.path] && (
-                          <span className="co-error">{formErrors[field.path]}</span>
-                        )}
-                      </div>
+          <hr className="pf-c-divider" />
+          {[...arrayFieldGroups].map((group) => {
+            const groupName = getGroupName(group, SpecCapability.arrayFieldGroup);
+            const fieldList = fields
+              .filter((f) => f.capabilities.includes(group))
+              .filter((f) => !_.isNil(inputFor(f)));
+
+            return (
+              <div id={group} key={group}>
+                <FieldGroup
+                  defaultExpand={
+                    !_.every(
+                      fieldList,
+                      (f) => f.capabilities.includes(SpecCapability.advanced) && !f.required,
+                    )
+                  }
+                  groupName={groupName}
+                >
+                  <div key={group} className="row">
+                    <label className="form-label">{groupName}</label>
+                    <div className="co-operand-field-group">
+                      {fieldList.map((field) => (
+                        <div key={field.path}>
+                          <div className="form-group co-create-operand__form-group">
+                            <label
+                              className={classNames('form-label', {
+                                'co-required': field.required,
+                              })}
+                              htmlFor={field.path}
+                            >
+                              {field.displayName}
+                            </label>
+                            {inputFor(field)}
+                            {field.description && (
+                              <span
+                                id={`${field.path}__description`}
+                                className="help-block text-muted"
```

Comment:
Not new, but is `text-muted` needed? I thought `help-block` already set the muted text color.